BlueDBInMemory/src/main/java/io/bluedb/memory/BlueDbInMemory.java

```diff
@@ -32,9 +32,18 @@ public BlueDbInMemory(Path directory) throws BlueDbException {
 		initializeFromDirectory();
 	}
 
+	@Override
+	public <T extends Serializable> BlueCollection<T> getCollection(String name) throws BlueDbException {
+		synchronized(collections) {
+			@SuppressWarnings("unchecked")
+			BlueCollection<T> collection = (BlueCollection<T>)(collections.get(name));
```

Comment:
Do we want to do this in a try and trow a BlueDbException with a caused by? Or do we want to add `throws ClassCastException` to the method signature?